disdrodb/tests/test_config_files.py

```diff
@@ -0,0 +1,336 @@
+import os
+import yaml
+from typing import Dict, List, Set, Union, Optional
+from pydantic import BaseModel, ValidationError
+import pytest
+
+
+# Define the pydantic models for *.bins.yaml config files
+class L0_data_format_2n_level(BaseModel):
+    n_digits: Optional[int]
+    n_characters: Optional[int]
+    n_decimals: Optional[int]
+    data_range: Optional[list]

```

Comment:
Maybe we need to check that the list length must be 2 (min and max vales).
And that data-range can also be `null` in the YAML?

Reply:
ok done via a pydantic validator